docs/data/joy/components/modal/modal.md

```diff
@@ -8,113 +8,139 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/dialog-modal/
 
 # Modal
 
-<p class="description">The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, or whatever else.</p>
-
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+<p class="description">The modal component provides a solid foundation for creating dialogs, popovers, lightboxes, and other interactive elements.</p>
 
 ## Introduction
 
-Joy UI provides three modal-related components:
+Modals are interactive dialog boxes or overlays that capture user inputs or present information.
+
+In Joy UI, modals are managed through three related components:
 
 - [`Modal`](#basic-usage): A container that renders its `children` node in front of a backdrop component.
 - [`ModalClose`](#dialog): A button for closing the modal.
 - [`ModalDialog`](#dialog): A component for rendering a modal dialog.
 
 {{"demo": "ModalUsage.js", "hideToolbar": true, "bg": "gradient"}}
 
-### Features
+The Modal component comes equipped with features designed to improve user interaction and accessibility:
 
-- 🥞 Manages modal stacking when more than one is needed.
-- 🪟 Automatically creates a backdrop element to disable interaction with the rest of the app.
-- 🔐 Disables page scrolling while open.
-- ⌨️ Manages focus correctly between the modal and its parent app.
-- ♿️ Adds the appropriate ARIA roles automatically.
+- Manages modal stacking when more than one is needed.
+- Automatically create a backdrop element to disable interactions with the rest of the app.
+- Disables page scrolling when open.
+- Manages focus between modal and its parent app.
+- Adds appropriate ARIA roles automatically.
```

Comment:
let's cut this section entirely - I found that we rarely have enough to say to justify a "Features" section on most pages, so I'd rather not have it than apply it inconsistently